### PR TITLE
this commit adds code for ticket 6 viewing comments for an article

### PR DESCRIPTION
**Pull request by hja7333**
Dave gave permission for this pr after 4pm today. Thanks Dave

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getComments } from "../utils/api";
+import styles from "../styles/Comments.module.css";
+const Comments = () => {
+  const [comments, setComments] = useState([]);
+  const { article_id } = useParams();
+  useEffect(() => {
+    getComments(article_id).then((data) => {
+      console.log(data.comments);
```
Comment (dav3rid) [inline src/components/Comments.jsx]: Can strip out this console log here

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      setComments(data.comments);
+    });
+  }, [article_id]);
+
```
Comment (dav3rid) [inline src/components/Comments.jsx]: You'll want to implement the `isLoading` pattern here - there is currently no indication to the user that comments are indeed loading

File: src/pages/SingleArticle.jsx [hunk 3]
```diff
@@ -31,12 +32,15 @@ const SingleArticle = () => {
         <img
           className={styles.image}
           src={article.article_img_url}
-          alt={`image of ${article.title}`}
+          alt={`${article.title}`}
```
Comment (dav3rid) [inline src/pages/SingleArticle.jsx:35]: this doesn't need to be in a template literal

File: src/utils/api.js [hunk 4]
```diff
@@ -1,15 +1,21 @@
 import axios from "axios";
-const articlesApi = axios.create({
+const newsApi = axios.create({
```
Comment (dav3rid) [inline src/utils/api.js:2]: Nice renaming - definitely clearer

**Review by dav3rid — CHANGES_REQUESTED** [review]
It does look a little janky in mobile view - you could allow the image for an article to spread out slightly more.

There are 2 errors I found regarding your css modules
- Don't use absolute paths as they rely on being run from your machine directly - use relative paths instead eg `import styles from "/Users/alhunt/northcoders/frontend/nc-news/src/styles/ArticleContainer.module.css";` should be `import styles from "../styles/ArticleContainer.module.css";`

File: src/pages/HomePage.jsx [hunk 5]
```diff
@@ -7,11 +7,12 @@ import TopicBar from "../components/TopicBar";
 import styles from "/Users/alhunt/northcoders/frontend/nc-news/src/styles/ArticleContainer.module.css";
```
Comment (JVal-NC) [inline src/pages/HomePage.jsx]: Can you fix this import to avoid importing from local files. New PR after please!

**Review by JVal-NC — COMMENTED** [review]
Hi Al, can you fix the import in the comments and do another PR, thanks!

**Review by dav3rid — APPROVED** [review]

**Pull request merged**